source/install/self-managed-changelog.md

```diff
@@ -90,6 +90,18 @@ Multiple setting options were added to ``config.json``. Below is a list of the a
 
+### Upcoming Deprecations and Breaking Changes in Mattermost v8.0
+
+The following deprecations and breaking changes are planned for the Mattermost v8.0 release, which is scheduled for 2023/06/16. This list is subject to change prior to the release.
+
+1. Update module version to v8.0. Clients will have to change their module path to the new structure. Very minimal change. There are no API level breaking changes that would require code changes.
+2. Remove ``ExperimentalSettings.PatchPluginsReactDOM``.
```

Comment:
I'm double checking the first point here, but here's some more information for this change. 
```
2. Remove ``ExperimentalSettings.PatchPluginsReactDOM``. If this setting was previously enabled, confirm that:
    1. All Mattermost-supported plugins are updated to the latest versions.
    2. Any other plugins have been updated to support React 17. See the [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html) for v7.7 for more information.
```
Also, I don't know if there's a better way to link directly to that note